script/licenses/license-overrides.ts

```diff
@@ -44,4 +44,23 @@ COPYRIGHT HOLDERS WILL NOT BE LIABLE FOR ANY DIRECT, INDIRECT, SPECIAL OR CONSEQ
 
 The name and trademarks of copyright holders may NOT be used in advertising or publicity pertaining to the work without specific, written prior permission. Title to copyright in this work will at all times remain with copyright holders.`,
   },
+
+  'tslib@2.0.0': {
+    repository: 'git+https://github.com/microsoft/tslib',
+    license: 'BSD',
```

Comment:
I had to add this because `tslib` has a 0-BSD license, but not sure (see https://github.com/microsoft/tslib/blob/master/LICENSE.txt), but not sure if we should instead add 0-BSD to the [`legal-eagle` package](https://github.com/atom/legal-eagle)